DependencyInjection/SonataUserExtension.php

```diff
@@ -52,7 +52,7 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('google_authenticator.xml');
         $loader->load('twig.xml');
 
-        if ('orm' === $config['manager_type'] && isset($bundles['FOSRestBundle']) && isset($bundles['NelmioApiDocBundle'])) {
+        if ('orm' === $config['manager_type'] && isset($bundles['FOSRestBundle']) && isset($bundles['NelmioApiDocBundle']) && isset($bundles['JMSSerializerBundle'])) {
```

Comment:
Can you use this as an opportunity to simplify this into : 

`isset(
    $bundles['FOSRestBundle'],
    $bundles['NelmioApiDocBundle'], 
    $bundles['JMSSerializerBundle']
)`